client/extensions/woocommerce/app/order/order-details/table.js

```diff
@@ -175,6 +240,13 @@ class OrderDetailsTable extends Component {
 							showTax={ showTax }
 						/>
 					) }
+					{ isEditing && (
```

Comment:
Thinking this should conditionally show also if no tax is collected for a store - like those in Oregon - it is showing for me right now... almost think we had a regression at some point too as each line item is showing tax again for me:

![order_details_ _care_for_animals_ _wordpress_com](https://user-images.githubusercontent.com/22080/31843549-7100a25e-b5a8-11e7-98d6-5634365d0cc9.png)

Reply:
Line items [always have tax](https://github.com/Automattic/wp-calypso/pull/16656#issuecomment-321333490), what we removed was the tax col from the totals area. I agree that this is not needed on the edit screen though.